src/lib/viewers/media/DashViewer.js

```diff
@@ -460,24 +460,22 @@ class DashViewer extends VideoBaseViewer {
     }
 
     /**
-     * Loads auto-generated captions
+     * Turn Box Skills transcript cards into Shaka Player text cues
      *
-     * @return {void}
+     * @param {Object} transcriptCards - Transcript cards
+     * @return {Array} text cues
      */
-    loadAutoGeneratedCaptions() {
-        const textCues = [];
-
-        // Convert Box Skill transcript cards to Shaka Text Cues
-        const skillsCards = getProp(this.options, 'file.metadata.global.boxSkillsCards.cards');
-        // No skills are available on the file
-        if (!skillsCards) {
-            return;
+    parseTextCues(transcriptCards) {
+        if (!transcriptCards) {
+            return [];
         }
 
-        const transcriptCard = skillsCards.find((card) => card.skill_card_type === 'transcript');
+        const textCues = [];
```

Comment:
Since `textCues` isn't being used until after the `if`, why not declare it there?